### PR TITLE
Resolve the temperature conversion test

### PR DESCRIPTION
**Pull request by salcudeann**

### CODE REVIEW

File: 01_temperature/temperature.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  if val==32
+    0
+  else
+    5.0*(val-32)/9.0
+  end
+=end
+end
+
+def ctof(val)
+  if val == 0
```
Comment (pirvudoru) [inline 01_temperature/temperature.rb:6]: How can we rewrite this as a one liner?

File: 01_temperature/temperature.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  val = val-32
+  5.0*val/9.0
+
+=begin
+  if val==32
+    0
+  else
+    5.0*(val-32)/9.0
+  end
+=end
```
Comment (pirvudoru) [inline 01_temperature/temperature.rb]: Learn about good comments: https://blog.submain.com/field-guide-code-comments/

**Review by pirvudoru — CHANGES_REQUESTED** [review]

File: 01_temperature/temperature.rb (hunk 1 above)
Comment (salcudeann) [reply 01_temperature/temperature.rb:15]: 5.0*(val-32)/9.0

File: 01_temperature/temperature.rb (hunk 2 above)
Comment (salcudeann) [reply 01_temperature/temperature.rb]: Got it!

File: 01_temperature/temperature.rb (hunk 1 above)
Comment (salcudeann) [reply 01_temperature/temperature.rb:6]: Else statement not needed, when _val_ is 0 the result will be 9.0*0/5.0+32 = 32.
So, the return of this method should be  9.0 * val/5.0 + 32

Comment (alex-damian-negru) [reply 01_temperature/temperature.rb:6]: We can also use `.zero?` method instead. https://ruby-doc.org/core-3.0.0/Integer.html#method-i-zero-3F.

Nice job noticing that the `if` isn't needed.

Tip: In Ruby, `if` statements can also be rewritten using the ternary operator. https://www.rubyguides.com/2019/10/ruby-ternary-operator/

**Comment by alex-damian-negru** [comment]
The changes here https://github.com/salcudeann/learn_ruby/pull/2#discussion_r669320100 weren't applied.

**Pull request merged**